crates/teloxide-core/src/types/chat_member.rs

```diff
@@ -354,7 +368,7 @@ impl ChatMemberKind {
         match self {
             Self::Owner(_) => true,
             Self::Administrator(Administrator { can_manage_chat, .. }) => *can_manage_chat,
-            Self::Member | Self::Restricted(_) | Self::Left | Self::Banned(_) => true,
+            Self::Member | Self::Restricted(_) | Self::Left | Self::Banned(_) => false,
```

Comment:
Hm, can members in "normal" groups/all members are admins = true view the log, etc? Do we need to add a mention of this in the docs?

Reply:
This `can_manage_chat` permission is not very clear to me. Apparently, this field is present only on `Administrator` chat member kind. According to my tests, the regular members can be granted at most `can_change_info`, where they still don't have access to event log, and I suppose everything else described in `can_manage_chat`.

I think think the current doc exhaustively specifies the algorithm of how this method resolves to `true` or `false`. Nothing to add on that regard 🤔